Clarify ReorderableListView drag handles on desktop and mobile

idkq opened the pull request:
Prevents this https://github.com/flutter/flutter/issues/80553

More info here https://github.com/flutter/flutter/pull/80576

Comments only change. These changes were pre-approved on the pull 80576 but not pushed.

idkq committed:
Update ReorderableListView docs to avoid confusion between desktop and mobile (v2)

Prevents this https://github.com/flutter/flutter/issues/80553

More info here https://github.com/flutter/flutter/pull/80576

Comments only change. These changes were pre-approved on the pull 80576 but not pushed.

Hixie commented on packages/flutter/lib/src/material/reorderable_list.dart:
```diff
@@ -29,6 +29,12 @@ import 'theme.dart';
+/// A drag handle is stacked over the center of edge on
+/// [TargetPlatformVariant.desktop] platforms. On [TargetPlatformVariant.mobile]
+/// platforms, no handle is visible and a long press anywhere on the item starts
+/// a drag. To change this behavior use
```
instead of "use" the Flutter voice would normally say "consider"

darrenaustin reviewed (changes requested):
Sorry for the delay on this review. Thanks for the contribution, this looks good. I agree with Ian that we should probably reword that to 'consider'. Thx.

idkq committed:
Using Flutter voice

Change 'use' for 'consider' https://github.com/flutter/flutter/pull/86737/files#r782619940

darrenaustin reviewed (changes requested):
@idkq thanks for the update. Unfortunately it looks like the analyzer didn't like the trailing space:

```
sdk/packages/flutter/lib/src/material/reorderable_list.dart:35: trailing U+0020 space character
```

https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8825148537102622513/+/u/Analyze/analyze/test_stdout

Can you fix that? Then I think we should be good to go. Thx.

darrenaustin commented:
Looks like it has been a while on this, so I just submitted these changes with some other doc fixes with #98954. Closing this PR. Thanks @idkq.
